internal/command/deploy/deploy.go

```diff
@@ -215,6 +215,19 @@ func New() *Command {
 			Description: "Do not run the release command during deployment.",
 			Default:     false,
 		},
+		flag.Bool{
+			Name:        "remote-deploy",
```

Comment:
Hi @rugwirobaker ! Here's trying my first pass on code reviewing a flyctl-deployments-related PR( let me know if this kind of review is useful or not! ): 

**Topic: I want to point out, in case this "remote-deploy" flag can be commented out, implemented, or of course, left as-is for future use**

1. **Is the flag "remote-deploy"  already used when running the deploy command?** The only code section I can find it mentioned in is this [commented out line](https://github.com/superfly/flyctl/pull/3769/files#diff-f1ed021d538250bb75f4922951ea5c467ce53115b46483d10c198a5c88c9cc60R617). Which points to the use of the function deployRemotely which I can't find declared anywhere just yet

2. **If there's no implementation yet for its use, does this flag need to be functional before this PR is merged?** It's not right, as this PR will just "provide the basis for remote deployers..."?

3. **If it's not yet needed, can this be commented out for now?** I think this can be added later on a separate PR, with an the implementation for its use. 

4. But! **Of course, it can also be left alone, as it is now, for future use.**

Reply:
Yeah, I commented it out that part but forgot to remove the flag. Thanks for pointing it out.